world_universities_and_domains.json

```diff
@@ -13431,11 +13431,11 @@
     "web_pages": [
       "http://www.wwc.edu/"
     ],
-    "name": "Walla Walla College",
+    "name": "Walla Walla University",
     "alpha_two_code": "US",
-    "state-province": null,
+    "state-province": "Washington",
     "domains": [
-      "wwc.edu"
+      "http://www.wallawalla.edu"
```

Comment:
Hi @thephred, the domain part should not contain http:// prefix. Also, can you please update web_pages as well?

Thank you